ow_regolith/include/ContactSensorPlugin.h

```diff
@@ -0,0 +1,52 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// ContactSensorPlugin is a Gazebo sensor plugin that publishes the names of any
+// links actively in contact with the collision object the sensor plugin has
+// been attached to.
+
+#ifndef TERRAIN_CONTACT_PLUGIN_H
+#define TERRAIN_CONTACT_PLUGIN_H
+
+#include <string>
+#include <set>
+#include <memory>
+#include <regex>
+
+#include <gazebo/gazebo.hh>
+#include <gazebo/common/common.hh>
+#include <gazebo/sensors/sensors.hh>
+
+#include <ros/ros.h>
+
+namespace ow_regolith {
+
+  class ContactSensorPlugin : public gazebo::SensorPlugin
+  {
+  public:
+    ContactSensorPlugin()
```

Comment:
My usual comment: be explicit about the remaining compiler-generated methods :)

Reply:
Done.